_data/food.csv

```diff
@@ -1,6 +1,7 @@
 date,vendor,time,site
-1/10,Barbirou Gyro,5-9pm,https://www.instagram.com/barbirou_greek_gyro_
-1/11,Mobile Chef,4-9PM,https://www.facebook.com/MobileChefCorp/
-1/12,Los Gallitos,4-9PM,https://www.instagram.com/losgallitospgh
-1/13,Polska Laska,5-10PM,https://www.instagram.com/polskalaskapgh/
+
 1/14,Veggies N'at,5-10PM,https://www.instagram.com/veggiesnat/
+1/15,Pure Grub,5-9PM,https://www.instagram.com/puregrub412/
+1/18,Mobile Chef,4-9PM,https://www.facebook.com/MobileChefCorp/
+1/19,El Rincon Oaxaqueno,4-9PM,https://www.instagram.com/elrinconoaxaqueno8/
```

Comment:
These are too long

Reply:
I'll show you how it looks

Reply:
<img width="335" alt="image" src="https://user-images.githubusercontent.com/842240/212406153-6ddc0cae-1fc5-422b-86c6-c49a9b3a90f5.png">